tests/dialects/test_dialect.py

```diff
@@ -1442,6 +1443,48 @@ def test_operators(self):
                 "spark": "FILTER(the_array, x -> x > 0)",
             },
         )
+        self.validate_all(
+            "a / b",
+            write={
+                "bigquery": "a / b",
+                "clickhouse": "a / b",
+                "hive": "a / b",
+                "mysql": "a / b",
+                "oracle": "a / b",
+                "postgres": "CAST(a AS DOUBLE PRECISION) / b",
+                "presto": "CAST(a AS DOUBLE) / b",
+                "snowflake": "a / b",
+                "sqlite": "CAST(a AS REAL) / b",
+            },
+        )
+
+    def test_div(self):
+        typed_div = exp.TypedDiv(this=exp.column("a"), expression=exp.column("b"))
```

Comment:
i guess we should add a new builder to support column // column !

Reply:
I think that'd be IntDiv, not TypedDiv?

Reply:
i guess that’s ok, python and sql are not exactly the same

Reply:
I think that'd be counter intuitive. 
Also, seems like `__floordiv__` already does IntDiv lol

Reply:
ah ok